README.md

```diff
@@ -84,7 +104,7 @@ end
 
 See the [example](examples/oneview_resource.rb) for more details.
 
-### oneview_ethernet_network
+#### oneview_ethernet_network
```

Comment:
I like the idea of organizing resources by type, but I don't think we can make the resource name headings any smaller than H3. When it's H4 like this, resources like [oneview_logical_interconnect_group](https://github.com/HewlettPackard/oneview-chef/blob/7c3b482d3667ad925c3172d245df751c9dbeaf3d/README.md#oneview_logical_interconnect_group) get pretty confusing, as some of the properties are the same size as the resource heading. You can always bump up the size of `Resources` and `Resources` to H1 and H2.

Or you can just remove the OneView & ICSP organizational headings. It will be pretty clear which are which.